frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions/ExpressionEditorSuggestions.tsx

```diff
@@ -1,5 +1,11 @@
 import cx from "classnames";
-import { useEffect, useRef, useCallback, type ReactNode } from "react";
+import {
```

Comment:
The dropdown does not disappear if I click a popular function suggestion.
It disappears after second click.


https://github.com/metabase/metabase/assets/6830683/29d71232-27a1-4e9c-b961-989704118a5d